src/components/SearchPage/SearchPageComponent.js

```diff
@@ -27,8 +33,11 @@ export function SearchPageComponent() {
       {results &&
         results.map(result => (
           <>
-            <h2>{result.name}</h2>
-            <h3>{result.mimeType}</h3>
+            <h2
+              style={{ 'border-top': '1px solid black' }}
+              onClick={() => handleClick(result)}>
```

Comment:
`result` is passed to the `handleClick`, but `handleClick` accepts `note`. That's confusing. I believe the `handleClick` should accept `searchResult`, not `note`.